File: 1 Revalida en python/5 Funciones/12.py
#funcion para coordenadas polares
import math
def complejo(a,b,n):
	rho=math.sqrt((a*a)+(b*b))
	ang=(math.asin((b/rho)))
	
	rhon=rho**n
	angn=n*ang
	
	an=rhon*(math.cos(angn))
	bn=rhon*(math.sin(angn))
	print an,bn
a=float(input("ingrese un numero"))
b=float(input("ingrese un numero"))
n=float(input("ingrese un numero"))
print complejo(a,b,n)

File: 2 Revalida en C++/5 Funciones/12.cpp
///funcion para coordenadas polares
#include <bits/stdc++.h>
using namespace std;
void complejo(float a,float b,float n)
{
    float rho,ang,rhon,angn,an,bn;
    rho=sqrt(pow(a,2)+pow(b,2));
	ang=asin((b/rho));

	rhon=pow(rho,n);
	angn=n*ang;
	an=rhon*cos(angn);
	bn=rhon*sin(angn);

	cout<<"las coordenadas polares son ="<<an<<bn;
}
int main()
{
    float a,b,n;
    cout<<"ingrese a:";
    cin>>a;
    cout<<"ingrese b:";
    cin>>b;
    cout<<"ingrese n:";
    cin>>n;
    complejo(a,b,n);
    return 0;
}
